app/views/sequencescape/api/v2/tubes/_tube.html.erb

```diff
@@ -0,0 +1,10 @@
+<li class="labware">
+  <%= link_to(tube, id: "tube_#{tube.barcode.ean13}", class: "list-group-item-action") do %>
+    <h3 class="barcode list-group-item-heading"><%= useful_barcode(tube.barcode) %> <%= state_badge(tube.state) %></h3>
+    <p><%= tube.name %>
+    <dl>
+      <dt>Created</dt><dd><%= tube.created_at.to_formatted_s(:date_created) %></dd>
+      <dt>Type</dt><dd><%= tube.purpose.name %></dd>
+    </dl>
+  <% end -%>
+</li>
```

Comment:
where will this be displayed?

Reply:
`app/views/tube_creation/pooled_tubes_from_whole_tubes.html.erb` renders the partial:
`app/views/search/_inbox.html.erb`
Which contains
`render sorted_labware`
When render receives an array, rails treats it as a collection, and selects the template to used based on the class name. (More accurately various helper methods provided by ActiveRecord/ActiveModel)
In this case we were using the default template for the V1 API stuff, and this just sets up a similar template for the V2 stuff, allowing rails to switch over automatically.

Reply:
I've not removed the V1 templates yet, as we'll need to migrate over the inboxes first. (Which we should definitely do)

Reply:
Cool OK thanks
Do you think it's worth a comment in this file with an abridged version of that? Just thinking if someone comes across it while on a deleting spree it's not going to be searchable where it's used